e2e-final/config/test-cases-block.yml

```diff
@@ -163,4 +163,20 @@
       iops: 2000
       tags: "Testing VPC volume from library with invalid custom iops"
     vpcZone: eu-gb-2
-  success: false
\ No newline at end of file
+  success: false
```

Comment:
As of now this looks good can we see if we can add negative test cases here ?

Reply:
https://github.ibm.com/alchemy-containers/armada-storage/issues/3831